include/fmt/chrono.h

```diff
@@ -680,7 +705,22 @@ FMT_CONSTEXPR const Char* parse_chrono_format(const Char* begin,
       continue;
     }
     if (begin != ptr) handler.on_text(begin, ptr);
-    ++ptr;  // consume '%'
+    c = *++ptr;  // consume '%'
+    if (ptr == end) FMT_THROW(format_error("invalid format"));
```

Comment:
Maybe these lines should be in a different order?
First checking and then access to the iterator.

Or if `ptr` - is always valid iterator:
```c++
    ++ptr;  // consume '%'
    if (ptr == end) FMT_THROW(format_error("invalid format"));
    c = *ptr;
```
?

Reply:
You are absolutely right! Thanks for pointing it out.